<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="scoreLayout">
     <item>
      <widget class="QLabel" name="playerScore">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Punkty Gracza: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="computerScore">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Punkty Komputera: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="gameLayout">
     <item>
      <widget class="QLabel" name="leftPhoto">
       <property name="text">
        <string>Photo 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item>
      <widget class="QLabel" name="rightPhoto">
       <property name="text">
        <string>Photo 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="resultLabel">
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>18</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Start!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="bottomLayout" stretch="0,0,0">
     <property name="spacing">
      <number>23</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="rockButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kamień</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paperButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Papier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scissorsButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nożyce</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
